src/papyrus-lang-vscode/resources/sse/tasks.json

```diff
@@ -0,0 +1,72 @@
+{
+    // created by vscode papyrus-lang papyrus.skyrimSpecialEdition.generateProject
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "label": "Compile Papyrus (All)",
+            "type": "process",
+            "command": "${config:papyrus.skyrimSpecialEdition.installPath}\\Papyrus Compiler\\PapyrusCompiler.exe",
```

Comment:
Keep in mind that if this setting is incorrect, we fall back to what is in the Windows registry. So, the server might be working even though this setting is actually wrong.

Reply:
has been removed now